Permissions/ViewController.m

```diff
@@ -167,24 +168,22 @@ - (void) rowTouchedBackgroundLocationServices {
 
 - (void) rowTouchedContacts {
   NSLog(@"Contacts requested");
-  ABAddressBookRef addressBook = ABAddressBookCreateWithOptions(NULL, NULL);
+  CNContactStore *addressBook = [[CNContactStore alloc] init];
 
   if (addressBook) {
-    self.addressBook = CFAutorelease(addressBook);
+    self.addressBook = addressBook;
 
      // Register for a callback if the addressbook data changes this is
      // important to be notified of new data when the user grants access to the
      // contacts. the application should also be able to handle a nil object
      // being returned as well if the user denies access to the address book.
-    ABAddressBookRegisterExternalChangeCallback(self.addressBook,
-                                                handleAddressBookChange,
-                                                (__bridge void *)(self));
+     // ABAddressBookRegisterExternalChangeCallback(self.addressBook,
```

Comment:
Could you please confirm that we have implemented the callback subscription in another place? If so - could you please update the comments and remove these lines?

Reply:
Implemented callback.